app/lib/common/dialogs/invite_to_room_dialog.dart

```diff
@@ -9,6 +9,18 @@ import 'package:atlas_icons/atlas_icons.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
+final userNameRegExp = RegExp(
```

Comment:
Would be handy if we put this in ```utils``` if we have more use-cases than here ;)

Reply:
fair... but do we have more use cases for this?!?